internal/npm_install/npm_install.bzl

```diff
@@ -267,6 +309,7 @@ cd "{root}" && "{npm}" {npm_args}
     )
     if result.return_code:
         fail("pre_process_package_json.js failed: \nSTDOUT:\n%s\nSTDERR:\n%s" % (result.stdout, result.stderr))
+    is_pnp = result.stdout.strip()
```

Comment:
It looks like there is a json parsing library for starlark: https://github.com/bazelbuild/bazel/issues/3732#issuecomment-502735610

Please use that instead of this, this is too hacky.